dotnet/SimpleSauce/SauceSession.cs

```diff
@@ -1,7 +1,50 @@
-﻿namespace SimpleSauce
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+
+namespace SimpleSauce
 {
     public class SauceSession
     {
+        public ChromeOptions ChromeOptions { get; private set; }
         public DataCenter DataCenter { get; set; } = DataCenter.UsWest;
+
+        private readonly IRemoteDriver _remoteDriverManager;
+        private readonly SauceOptions options;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field 'options' or refactor the code to use its value.](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4471446)